php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S5328.html

```diff
@@ -0,0 +1,36 @@
+<p>Setting session IDs is security-sensitive. Dynamically setting session IDs with client-supplied data or insecure hashes may lead to session
+fixation attacks and may allow an attacker to hijack another user's session.</p>
+<h2>Ask Yourself Whether</h2>
+<ul>
+  <li> the session ID is hardcoded </li>
```

Comment:
```suggestion
```
I think this is a leftover from the first pull request. This can probably be removed as it doesn't make much sense. But instead maybe we can warn about session IDs being set from a `$_GET` variable (query string), as I've actually seen this happen a few times in the past

Reply:
The case of `$_GET` variable injected in `session_id` will be handled by the Injection Vulnerability rule we will probably implement later.

Reply:
@agigleux Ah okay, sounds good!